src/platform_impl/windows/window.rs

```diff
@@ -1398,7 +1387,7 @@ unsafe fn init(
     let menu = attributes.platform_specific.menu;
     let mut initdata = InitData {
         event_loop,
-        attributes,
+        attributes: attributes.clone(),
```

Comment:
Hm, could you stash the fullscreen/maximized and then use them instead of cloning the attributes state?

Reply:
I fixed this.